cmd/zt_scenario_helpers_for_test.go

```diff
@@ -175,6 +175,104 @@ func (scenarioHelper) generateBlobsFromList(c *chk.C, containerURL azblob.Contai
 	time.Sleep(time.Millisecond * 1500)
 }
 
+func (scenarioHelper) generateBlobsFromListForPageCopy(c *chk.C, containerURL azblob.ContainerURL, blobList []string) {
```

Comment:
I thought there's already a method that does this? e.g. generateBlobsFromList. You could rename it to generateBlockBlobsFromList, and take in the data to upload (`ForPageCopy` because of the data size right?).

Reply:
Good point. I think I was just in the wrong mindset when writing these tests and ended up duplicating a lot of code.